Nos/Models/URLParser.swift

```diff
@@ -103,8 +102,8 @@ struct URLParser {
             for match in matches {
                 let nip05Range = Range(match.range(withName: "nip05"), in: string)
                 if let nip05Range {
-                    let nip05 = String(string[nip05Range])
-                    let webLink = "https://njump.me/\(nip05)"
+                    let nip05 = String(string[nip05Range]).lowercased()
+                    let webLink = "@\(nip05)"
```

Comment:
Again, might be nice to refactor some of this out to another private func due to the layers of nesting. Something like `replaceNip05(nip05Range)`, maybe?

Reply:
I refactored this in a new function, but to be honest I don't think we can do much more here, everything is interconnected because it depends on the range, the specific regular expression, the exact mutable string in which that regex was run in order to be sure that nothing fails. And I don't want to instantiate a new regular expression due to a potential performance penalty as I think regex are heavy. So the result is a more clear function named `replaceNIP05IfNeeded` but in practice it uses all the variables we instantiated just before.